<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>216</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>216</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>216</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fixing the Trains</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="speed">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>55</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="PPM">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>270</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>420</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="door_lock">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PPM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors locked</string>
    </property>
   </widget>
   <widget class="QComboBox" name="door_open">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="door_free">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors free</string>
    </property>
   </widget>
   <widget class="QComboBox" name="station">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>At station</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_val">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>114</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency brake</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors locked</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors open</string>
    </property>
   </widget>
   <widget class="QLabel" name="brake_out">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="speed_out">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="open_out">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lock_out">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="new_train_list"/>
    </widget>
    <addaction name="menuNew"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="new_train_list">
   <property name="text">
    <string>Training list</string>
   </property>
  </action>
  <action name="Train_Network">
   <property name="text">
    <string>Train Network</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>speed</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>speed_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
